clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -52,34 +52,6 @@ const ConsentBanner: FunctionComponent<BannerProps> = ({
     };
   }, []);
 
-  // add listeners for ESC and clicking outside of component
-  useEffect(() => {
-    const handleClickOutside = (event: MouseEvent) => {
-      if (ref.current && !ref.current.contains(event.target as Node)) {
```

Comment:
we can also remove `const ref = useRef<HTMLDivElement>(null);` from line 38 and all usages / imports related to it